pkg/util/patroni/patroni.go

```diff
@@ -74,6 +69,28 @@ func (p *Patroni) Failover(master *v1.Pod, candidate string) error {
 
 		return fmt.Errorf("patroni returned '%s'", string(bodyBytes))
 	}
+	return nil
+}
+
+// Failover does manual failover via Patroni POST API call
```

Comment:
Manual as in "make Patroni do that " ?

Reply:
yes, 'manual' might be not the best name here.